core/templates/pages/terms-page/terms-page.component.html

```diff
@@ -40,7 +40,7 @@ <h2>Privacy Policy</h2>
     <p>
       For more information about how we use your personal information and what
       types of information we collect and why, please refer to our
-      <a href="/privacy-policy"
+      <a href="/privacy-policy "
```

Comment:
There shouldn't be a space before the closing quote, I think. (You can check for these things by auditing the changes in the GitHub "Files Changed" tab before sending the PR for review.)